### PR TITLE
[chassis] skip test_route_flow_counter.py for t2

### PR DESCRIPTION
**Pull request by wenyiz2021**
### Description of PR

PR https://github.com/sonic-net/sonic-mgmt/pull/8732 added test topo to `any`, but the CLI this test used will not work for multi-asic card, also it should not run on SUP.
Before we make this test to MA workable, we should not run on T2.
Summary:
Fixes # (issue) https://github.com/sonic-net/sonic-mgmt/issues/10208
https://github.com/sonic-net/sonic-mgmt/issues/10203

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### How did you verify/test it?
run on T2
```
=========================== short test summary info ============================
SKIPPED [9] /var/src/private/sonic-mgmt-int/tests/flow_counter/flow_counter_utils.py:119: Skip test for T2 and multi-asic
========================= 9 skipped in 360.47 seconds ==========================
```

### CODE REVIEW

File: tests/route/test_route_flow_counter.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 allure.logger = logger
 
 pytestmark = [
-    pytest.mark.topology("any")
+    pytest.mark.topology('t0', 't1', 'm0', 'mx'),
```
Comment (SuvarnaMeenakshi) [inline tests/route/test_route_flow_counter.py:13]: better to skip for t2 specifically as "any" will include all topologies like dual tor or t1-lag etc

Comment (abdosi) [reply tests/route/test_route_flow_counter.py:13]: isntead of making this change can we update api `def get_route_flow_counter_capability(dut, route_flow_counter_capability):` return False . We can check either multi-asic or top as t2

Comment (wenyiz2021) [reply tests/route/test_route_flow_counter.py:13]: @abdosi don't think we should return False if T2/MA, if so the test will always fail for T2

```
    if not wait_until(CAPABILITY_WAIT_TIME_IN_SEC, CAPABILITY_CHECK_INTERVAL_IN_SEC, 0,
                      get_route_flow_counter_capability, rand_selected_dut, route_flow_counter_capability):
        pytest_assert(False, 'Failed to get route flow counter capability')
```

**Review by abdosi — APPROVED** [review]

**Comment by wenyiz2021** [comment]
hi @wangxin , could you help to backport this PR to 202205 and 202305? thanks!

**Pull request merged**